src/styles/tools/_mixins.scss

```diff
@@ -4,6 +4,10 @@ $breakpoint-tablet: 820px;
   background: url("../../../public/images/background_image.png");
   background-size: 100%;
 
+  @media screen and (prefers-color-scheme: dark) {
+    background: url("../../../public/background_image_dark.png");
+  }
+
```

Comment:
Right now, that new `background_image_dark` is outside of the images folder. We should probably move it, which means we'll have to update this path. 

Also- are you getting this funkiness at super giant screen sizes? Can you see the hole in the tiling? 
<img width="357" alt="Screen Shot 2022-11-08 at 4 30 17 PM" src="https://user-images.githubusercontent.com/69602589/200680341-53edbbbb-d31e-425f-94f3-c351ff067a6f.png">

Reply:
Also, I exported that background-image from Figma once more, as an SVG, and that seemed to help the "busy-ness" that the current image has. It seems to match up better with the light-mode background now. Not sure what difference that made, besides it being an SVG extension. 

```
@media (prefers-color-scheme: dark) {
    background: url("../../../public/images/background_image.svg");
  }
```
<img width="598" alt="Screen Shot 2022-11-08 at 4 41 55 PM" src="https://user-images.githubusercontent.com/69602589/200681901-04f17216-8c61-4fec-852d-a71f1189e28c.png">
<img width="548" alt="Screen Shot 2022-11-08 at 4 42 05 PM" src="https://user-images.githubusercontent.com/69602589/200681923-ebb9c672-8ec3-4123-8a2a-ea59364bceff.png">

Reply:
OK, I am not sure why it wasn't PNG to begin with but it's much better and no weird gap so I'll make sure to download as PNG instead. Thanks for pointing out this issue and a fix! 🥇